zdm/iteration.py

```diff
@@ -848,8 +856,15 @@ def cube_likelihoods(grids,surveys,psetmins,psetmaxes,npoints,run,howmany,outfil
 	
 	# this is the order of fastest to slowest, i.e. we update index
 	# order[0] first, then order [1], etc
-	
-	order=[7,4,5,6,0,1,2,3]
+	# the order can depend on the methods being used
+	# note: takes this from the first grid, it should be identical for all grids
+	if grids[0].alpha_method==0:
```

Comment:
would be great to move from numeric indices for the parameters to string (i.e. easily interpretable).  Just raising this point here, not suggesting we do it now.

Reply:
I would have thought perhaps a dict? e.g. params["Emin"]? But in future, I want to divide this up "parameters" into different groups. One defining the FRB luminosity function, since some LFs will need different params than just Emin, Emax, gamma (e.g. Ebreak1,gamma1, Ebreak2,gamma2 etc); we currently have two params defining the host galaxy contribution, but this could be generalised; etc.

Reply:
I have decided this is a future to-do, and may get messy, so Im leaving this for now, and will work on it once Esan has a stable version going.